src/components/analyser/search/SearchOption.tsx

```diff
@@ -0,0 +1,47 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+import { Link } from '@reach/router';
+import React from 'react';
+import * as API from '../../../analyser/APIClient';
+
+export type SearchOptionProps = {
+  stock: API.Stock;
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    navlink: {
+      color: theme.palette.primary.main,
+      textDecoration: 'none',
+      textTransform: 'none',
+      textAlign: 'left',
+      padding: '8px 10px 8px 10px',
+      width: 100,
+      fontSize: theme.typography.body1.fontSize,
+    },
+  })
+);
```

Comment:
List is not styled correctly...
- should not display all stocks when clicking on the search bar (we handle thousands of stocks)
- names are not formatted correctls (see IBM), maybe use symbol instead? See how other websites did it..
<img width="1751" alt="image" src="https://user-images.githubusercontent.com/29731862/113297010-6b9f1880-92fa-11eb-98f6-9afce9083b5a.png">